spec/Section 2 -- Language.md

```diff
@@ -166,9 +166,9 @@ characters to support interoperation with as many other systems as possible.
 
 ## Query Document
 
-Document : Definition+
+Document :: Definition+
```

Comment:
In this case, it should be referenced here: https://github.com/facebook/graphql/blame/master/spec/Section%202%20--%20Language.md#L8-L11
Because right now it very confusing since above text mentioning only `::` and not `:`